packages/block-library/src/navigation/index.php

```diff
@@ -5,6 +5,659 @@
  * @package WordPress
  */
 
+/**
+ * Helper functions used to render the navigation block.
+ */
+class WP_Navigation_Block_Renderer_Gutenberg {
```

Comment:
Should we keep the name as is and include the webpack change instead?

Reply:
I thought this was simpler

Reply:
As soon as we backport this file to WP core, then it will throw a fatal error because of the class redeclaration. That's why I think that following the recommendation from Riad makes perfect sense.

Reply:
Ok I'll do that :)

Reply:
This file won't be backported to Core though

Reply:
It isn't targeting `trunk`, interesting :)